relay-general/src/store/normalize.rs

```diff
@@ -221,12 +221,69 @@ fn normalize_breakdowns(event: &mut Event, breakdowns_config: Option<&Breakdowns
     }
 }
 
-/// Ensure measurements interface is only present for transaction events
+/// Returns the unit of the provided metric.
+///
+/// For known measurements, this returns `Some(MetricUnit)`, which can also include
+/// `Some(MetricUnit::None)`. For unknown measurement names, this returns `None`.
+fn get_metric_measurement_unit(measurement_name: &str) -> Option<MetricUnit> {
+    match measurement_name {
+        // Web
+        "fcp" => Some(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        "lcp" => Some(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        "fid" => Some(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        "fp" => Some(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        "ttfb" => Some(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        "ttfb.requesttime" => Some(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        "cls" => Some(MetricUnit::None),
+
+        // Mobile
+        "app_start_cold" => Some(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        "app_start_warm" => Some(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        "frames_total" => Some(MetricUnit::None),
+        "frames_slow" => Some(MetricUnit::None),
+        "frames_slow_rate" => Some(MetricUnit::Fraction(FractionUnit::Ratio)),
+        "frames_frozen" => Some(MetricUnit::None),
+        "frames_frozen_rate" => Some(MetricUnit::Fraction(FractionUnit::Ratio)),
+
+        // React-Native
+        "stall_count" => Some(MetricUnit::None),
+        "stall_total_time" => Some(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        "stall_longest_time" => Some(MetricUnit::Duration(DurationUnit::MilliSecond)),
+        "stall_percentage" => Some(MetricUnit::Fraction(FractionUnit::Ratio)),
+
+        // Default
+        _ => None,
+    }
+}
+
+fn normalize_units(measurements: &mut Measurements) {
+    for (name, measurement) in measurements.iter_mut() {
+        let measurement = match measurement.value_mut() {
+            Some(m) => m,
+            None => continue,
+        };
+
+        let stated_unit = measurement.unit.value().copied();
+        let default_unit = get_metric_measurement_unit(name);
+        if let (Some(default), Some(stated)) = (default_unit, stated_unit) {
```

Comment:
It seems `default` is now a keyword in Rust. Although still experimental, see https://github.com/rust-lang/rfcs/blob/master/text/1210-impl-specialization.md. Can we use another name? 
```suggestion
        if let (Some(expected), Some(actual)) = (default_unit, stated_unit) {
```

Reply:
TIL, thanks!